apis/serving/v1beta1/inferenceservice_types.go

```diff
@@ -97,6 +97,25 @@ type PredictorExtensionSpec struct {
 	StorageURI *string `json:"storageUri,omitempty"`
 	// +optional
 	RuntimeVersion *string `json:"runtimeVersion,omitempty"`
+	// Storage Spec for model location
+	// +optional
+	Storage *StorageSpec `json:"storage,omitempty"`
+}
+
+type StorageSpec struct {
```

Comment:
Can we discuss this? we do not want to maintain two versions of InferenceService